### ISSUE TITLE
[C#][aspnetcore] Swashbuckle.AspNetCore should be used instead of Swashbuckle for .NET 4.6

### ISSUE THREAD

**Issue by mandrean** [swagger-api/swagger-codegen#5255]
##### Description

The ASP.NET Core server generator should use the new Swashbuckle for ASP.NET Core instead of the old Swashbuckle for .NET 4.6

From the developers of Swashbuckle:
```
Looking to use Swashbuckle with ASP.NET Core???|

This project and corresponding Nuget packages are specifically for ASP.NET Web API applications. If you're looking to use Swashbuckle with ASP.NET Core, checkout https://github.com/domaindrivendev/Swashbuckle.AspNetCore.|
```

##### Swagger-codegen version

`v2.3.0` or latest `master`, doesn't matter.

##### Swagger declaration file content or url

`sh ./bin/aspnetcore-petstore-server.sh` or your own `Swagger.json`, doesn't matter.

##### Steps to reproduce

0. `git clone https://github.com/swagger-api/swagger-codegen && cd swagger-codegen`
1. `git checkout origin 2.3.0 && git pull origin 2.3.0 --rebase`
2. `mvn clean package`
3. `sh ./bin/aspnetcore-petstore-server.sh`
4. `cd samples/server/petstore/aspnetcore/src/IO.Swagger`
5. `sh build.sh`
6. Check `swagger-codegen/samples/server/petstore/aspnetcore/src/IO.Swagger/project.json`

##### Suggest a Fix

Replace `Swashbuckle 6.0.0-beta901` with `Swashbuckle.AspNetCore 1.0.0-rc3` and update dependency imports, `Startup.cs` etc. according to needs.

Also, according to [Semver](http://semver.org/#spec-item-11), `beta < rc`.

**Comment by mandrean** [comment]
Fixed in #5257.

**Comment by wing328** [comment]
Closed via #5257
